config/routes.rb

```diff
@@ -71,89 +83,155 @@
           end
         end
       end
-      resources :events, only: %i[index create update new edit]
-      resources :feedback_messages, only: %i[index show]
-      resources :invitations, only: %i[index new create destroy]
-      resources :pages, only: %i[index new create edit update destroy]
-      resources :mods, only: %i[index update]
-      resources :moderator_actions, only: %i[index]
-      resources :navigation_links, only: %i[index update create destroy]
-      resources :privileged_reactions, only: %i[index]
-      resources :permissions, only: %i[index]
-      resources :podcasts, only: %i[index edit update destroy] do
-        member do
-          post :fetch
-          post :add_owner
-        end
-      end
 
       # NOTE: @citizen428 The next two resources have a temporary constraint
       # while profile generalization is still WIP
       constraints(->(_request) { FeatureFlag.enabled?(:profile_admin) }) do
         resources :profile_field_groups, only: %i[update create destroy]
         resources :profile_fields, only: %i[index update create destroy]
       end
-      resources :reactions, only: [:update]
-      resources :response_templates, only: %i[index new edit create update destroy]
+
+      # These redirects serve as a safegaurd to prevent 404s for any Admins
```

Comment:
I'm not sure was there an agreement for how long should these stay in? Are they forever routes?

Reply:
Not sure, being admin routes I think we can eventually remove them though. 

ps. tiny typo in `safeguard`

Reply:
lol oops 😅